### PR TITLE
Add features and known issues doc

### PR DESCRIPTION
**Pull request by CasperMok**
- Add feature list section in README.md
- Add a KNOWN-ISSUES.md

### CODE REVIEW

**Review by aravind-raveendran — DISMISSED** [review]

**Review by RameshDPrasad — DISMISSED** [review]

**Review by ldanilina — DISMISSED** [review]

File: KNOWN-ISSUES.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# v1.0-rc3
+
+The issues are mainly in the following areas
+
+* No proper background support on both Android and iOS.
+* Edges cases are not handled properly, such as mic/camera permission and no network.
+* Sometimes camera and mic do not work after re-joining the meeting.
+
+Not all the issues are listed there. They include
```
Comment (tarunkgarg) [inline KNOWN-ISSUES.md:9]: this can be rephrase to :
Not all the issues are listed here.

File: KNOWN-ISSUES.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+* Individual device or model-specified issues
```
Comment (tarunkgarg) [inline KNOWN-ISSUES.md:11]: change model-specified to  model-specific

File: KNOWN-ISSUES.md (hunk 1 above)
Comment (CasperMok) [reply KNOWN-ISSUES.md]: Done

File: KNOWN-ISSUES.md (hunk 2 above)
Comment (CasperMok) [reply KNOWN-ISSUES.md]: Done

**Review by tarunkgarg — APPROVED** [review]

**Review by aravind-raveendran — DISMISSED** [review]

**Review by tarunkgarg — APPROVED** [review]

**Review by aravind-raveendran — APPROVED** [review]

**Pull request merged**